src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -295,7 +295,9 @@ void SubgraphBaseTest::compile_model() {
     auto start_time = std::chrono::system_clock::now();
 
     configure_model();
-    core_configuration(this);
+    if (do_core_config) {
+        core_configuration(this);
+    }
```

Comment:
Instead of modifying the common part, we better modify the cpu specific `core_configuration` implementation with additional check for `inference_precision == element::undefined`